### PR TITLE
aws-pgd-3-nodes: updates

### PR DESCRIPTION
**Pull request by mw2q**
A number of updates primarily revolving around using edb-ansible to build a PGD cluster instead of TPA:

* Use edb-ansible instead of TPA to build PGD cluster
* Use "pgd" instead of "bdr" where appropriate to update terminology
* Update inventory template for edb-ansible PGD usage
* Update deploy playbook to use edb-ansible to setup and configure PGD cluster

These are some generic configuration updates:

* Update TPCC_WAREHOUSE default to 1000
* Update PG_INSTANCE_TYPE default to c5d.18xlarge
* Use gp3 instead of gp2 as more cost effective storage type, where 1000GB with 3000 IOPS is the max of both in that price range which also lets us handle a crash dump if need be
* Remove ansible async/poll usage, they are masking issues and giving false positives in failing tasks
* set -x in exec-hdb-rampup.sh just in case debugging is needed after the fact, and don't remove dynamically generated tcl scripts
* Use edb-ansible hammerdb roles to setup HammerDB

### CODE REVIEW

File: aws-pgd-3-nodes/vars.yml [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-bdr_dbname: edb
+pg_login_unix_socket: /var/run/edb/as{{ pg_version }}
 bdr_writers: 16
 bdr_wal_decoder: false
+pg_type: EPAS
+pg_database: edb
+enable_edb_repo: true
+install_pgd: true
+pgd_version: 5
+pgd_cluster_database: tpcc
```
Comment (bryan-bar) [inline aws-pgd-3-nodes/vars.yml:19]:
```suggestion
benchmark_database: tpcc
```

File: aws-pgd-3-nodes/vars.yml [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 hammerdb_version: 4.6
-hammerdb_install_path: "/opt/HammerDB-{{ hammerdb_version }}"
-hammerdb_data_path: /opt/hammerdb-data
-pg_login_unix_socket: /tmp
-bdr_dbname: edb
+pg_login_unix_socket: /var/run/edb/as{{ pg_version }}
 bdr_writers: 16
 bdr_wal_decoder: false
+pg_type: EPAS
+pg_database: edb
```
Comment (bryan-bar) [inline aws-pgd-3-nodes/vars.yml:15]:
```suggestion
pg_database: {% if pg_type=='EPAS' %}edb{% else %}postgres{% endif %}
```

**Review by bryan-bar — APPROVED** [review]
LGTM.

I included 2 suggestions for `vars.yml`. We can keep two variable names for all benchmarks when referencing the initial database and the benchmark/target database.

File: aws-pgd-3-nodes/vars.yml (hunk 1 above)
Comment (mw2q) [reply aws-pgd-3-nodes/vars.yml:19]: `pgd_cluster_database` also tells `setup_pdg` which database to initialize as the distributed database (if you ignore what we do to it in the prepare step).  i'm not sure if we benefit having both variables.

File: aws-pgd-3-nodes/vars.yml (hunk 2 above)
Comment (mw2q) [reply aws-pgd-3-nodes/vars.yml:15]: ok, made change, forced push.

**Pull request merged**